### DISCUSSION TITLE
Customize fzf preview window

### DISCUSSION THREAD

**Discussion by lcmen** [zk-org/zk-nvim#131] [Ideas]
Hey, I'm really sorry to bother you again! I just wanted to ask if it's possible to re-use `fzf-preview` preview command from `.zk/config.toml` when displaying notes with `ZkNotes` command using `fzf`?

I also use [fzf.vim](https://github.com/junegunn/fzf.vim) but for some reason built-in bindings to open files in new tab, split and vsplit (`CTRL-T` / `CTRL-X` / `CTRL-V`) don't seem to work. Any idea what I need to configure?

**Comment by karimknaebel** [comment]
the bindings are a feature of the pickers in `fzf.vim`, they are not shared with this plugin.

it's not possible right now to reuse the fzf-preview command in `.zk/config.toml`.

sorry!

FYI the fzf picker is implemented here: https://github.com/mickael-menu/zk-nvim/blob/main/lua/zk/pickers/fzf.lua

**Comment by mickael-menu** [comment]
Eventually I'd like to expose the `zk` configuration options through the LSP server.

**Comment by lcmen** [comment]
> Eventually I'd like to expose the `zk` configuration options through the LSP server.

That would be great :)
